README.md

```diff
@@ -55,6 +55,20 @@ To install Spectator Sport in your Rails application:
           <script defer src="/spectator_sport/events.js"></script>
         </head>
         ```
+    - Optional: If you would like to capture referrer and landing path data, add this to your `ApplicationController`:
+        ```rb
+        class ApplicationController < ActionController::Base
+          before_action :store_referrer_and_path
+
+          private
+
+          def store_referrer_and_path
+            session["referrer"] ||= request.referrer.to_s
+            session["landing_path"] ||= request.path.to_s
+          end
+        end
```

Comment:
I'd like to make this more contained, like:

```ruby
include SpectatorSport::ControllerSetup
```

(I'm not happy with that name; naming things is hard)

I also think that Spectator Sport may need its cookie, because the parent application could already be using those values.

Reply:
I can get behind long term thinking any day!